registration/forms.py

```diff
@@ -195,7 +221,8 @@ def clean_password(self):
         Check password strength.
         """
         password = self.cleaned_data.get('password')
-        if password and password_strength(password)['score'] < 2:
+        password_strength = self.cleaned_data.get('password_strength')
+        if password and self._check_password_strength(password_strength) is not None and password_strength < 3:
```

Comment:
Do we need to be that strict here? According to the docs:

>   2 # somewhat guessable: protection from unthrottled online attacks. (guesses < 10^8)
>   3 # safely unguessable: moderate protection from offline slow-hash scenario. (guesses < 10^10)

Can we settle for 2? I feel that a lot of people's passwords will be rejected otherwise.